src/QueryParserCustomFields.php

```diff
@@ -169,43 +170,51 @@ public function request(): array
         if (array_key_exists('sort', $this->request)) {
             $sort = $this->request['sort'];
 
-            if (!empty($sort)) {
+            if (!empty($sort) && strpos($sort, '|') !== false) {
```

Comment:
This will only allow to sort when there's a `|` on the string. Sorting without specifying the order wouldn't work, e.g. `?sort=name`.